alodataset/coco_base_dataset.py

```diff
@@ -341,7 +355,12 @@ def __call__(self, image, target):
 
 
 if __name__ == "__main__":
-    coco_dataset = CocoBaseDataset(sample=True)
+    coco_dataset = CocoBaseDataset(sample=False,img_folder = "test2017")
```

Comment:
Coding convention:
- A comma "," should be followed by a space
- There are no space before and after "="